modules/arm_plugin/src/transformations/arm_optimizations.cpp

```diff
@@ -309,19 +310,34 @@ bool ArmPlugin::pass::ArmOptimizations::run_on_function(std::shared_ptr<ngraph::
         manager.run_passes(f);
     }
 
-
     if (quantized) {
         Dump(f, "before_arm");
-        ngraph::pass::Manager manager;
-        manager.register_pass<pass::ConvolutionQuantizeFusion>();
-        manager.register_pass<pass::MeanQuantizeFusion>();
-        manager.register_pass<pass::DequantizeInputFusion>();
-        manager.register_pass<pass::AddDequantizeOnInputs>();
-        manager.register_pass<ngraph::pass::ConstantFolding>();
-        manager.register_pass<pass::ConvertQuantize>();
-        manager.register_pass<pass::ConvertBiasToI32>();
-        manager.register_pass<ngraph::pass::ConstantFolding>();
-        manager.run_passes(f);
+        {
+            ov::pass::Manager manager;
+            auto pass = manager.register_pass<ov::pass::GraphRewrite>();
+            pass->add_matcher<pass::ConvolutionQuantizeFusion>();
+            pass->add_matcher<pass::MeanQuantizeFusion>();
+            manager.run_passes(f);
+        }
+        {
+            ov::pass::Manager manager;
```

Comment:
I think it would be better to use single `Manager` for whole arm quantization processing and execute `run_passes` once